core/hydromet_JSON_to_DSS.py

```diff
@@ -72,12 +79,21 @@ def main(forcing_dir: plib, outputs_dir: plib, bin_dir: plib, filename: str,
                 if dur not in all_durations:
                     all_durations.append(dur)
                     dss_map(outputs_dir, variable, tstep, tstep_units, units,
-                                 data_type, to_dss = to_dss, open_op = 'a+')
+                                     data_type, to_dss = to_dss, open_op = 'a+')
                     temp_files.append(to_dss)
-                excess_df_to_input(outputs_dir, df, tstep, tstep_units, 
-                                                    scen_name, 'a+', to_dss) 
-    make_dss_file(outputs_dir, bin_dir, filename, remove_temp_files = False, 
+                    excess_df_to_input(outputs_dir, df, tstep, tstep_units, 
+                                                        scen_name, 'a+', to_dss) 
+
+            if scaling is True:
+                make_dss_file(outputs_dir, bin_dir, '{0}_{1}'.format(filename,dur), remove_temp_files = False, 
+                                                display_print = display_print)
+                temp_files.remove(to_dss)
```

Comment:
Not sure why you remove the to_dss from the list of temp_files, but then delete the to_dss in the next line when this is all taken care of in line 99.